src/sorcha/modules/PPCommandLineParser.py

```diff
@@ -110,4 +110,9 @@ def PPCommandLineParser(args):
             )
         )
 
+    # If verbose logging is on, log all the command line settings to INFO.
```

Comment:
It would be good if we change this so it's by default even if verbose logging is not on to ensure the user knows what was used with the output in the directory with their log file

Reply:
Sounds good. I made that change.